### ISSUE TITLE
Uncaught Error: 'target' is a required option

### ISSUE THREAD

**Issue by amit777** [framework7io/framework7#3843]
* Framework7 version: 6.0.12
* Svelte version: 3.35.0
* Platform and Target: Chrome browser
* easy to reproduce from CLI

### Describe the bug
When setting the "dev" compilerOptions for svelte-loader, the following error occurs:

```

  | SvelteComponentDev | @ | app.js_+_398_modules:48997
-- | -- | -- | --
  | Home | @ | app.js_+_398_modules:102322
  | view_create_default_slot | @ | app.js_+_398_modules:99961
  | create_slot | @ | app.js_+_398_modules:46986
  | router_context_provider_create_fragment | @ | app.js_+_398_modules:91258
  | init | @ | app.js_+_398_modules:48717
  | Router_context_provider | @ | app.js_+_398_modules:91311
  | view_create_each_block | @ | app.js_+_398_modules:100028
  | view_create_fragment | @ | app.js_+_398_modules:100091
  | init | @ | app.js_+_398_modules:48717
  | view_View | @ | app.js_+_398_modules:100411
  | app_svelte_create_default_slot | @ | app.js_+_398_modules:107725
  | create_slot | @ | app.js_+_398_modules:46986
  | app_create_fragment | @ | app.js_+_398_modules:50676
  | init | @ | app.js_+_398_modules:48717
  | App | @ | app.js_+_398_modules:50780
  | app_svelte_create_fragment | @ | app.js_+_398_modules:107861
  | init | @ | app.js_+_398_modules:48717
  | App_1 | @ | app.js_+_398_modules:108013
  | eval | @ | app.js_+_398_modules:108040
  | ./src/js/app.js | @ | app.js:41
  | __webpack_require__ | @ | app.js:579
  | (anonymous) | @ | app.js:1716
  | (anonymous) | @ | app.js:1720


```

### To Reproduce
1. `frameworks7 create`  (basic single view app with svelte)
2. modify ```build/webpack.config.js``` to include compilerOptions like this:
```
{
        test: /\.svelte$/,
        use: {
          loader: 'svelte-loader',
          options: {
            emitCss: true,
            compilerOptions: {
              dev: true
            }
          },
        },
      },
```
3. run ```npm start```

### Expected behavior
No error should occur

### Actual Behavior
the above error occurs

### Screenshots
![Screen Shot 2021-03-22 at 10 13 00 AM](https://user-images.githubusercontent.com/2703309/112030311-628e9a00-8af7-11eb-82b8-796463827054.png)

### Additional context
I found the error because we work with rollup.js, however I was able to reproduce it with a clean project generated with framework7 CLI and webpack.  I have a feeling this has to do with the way views are handled with svelte and f7.

**Comment by angrytongan** [comment]
This also occurs when following the [Initialize App](https://framework7.io/svelte/init-app.html) instructions using the default [svelte template](https://github.com/sveltejs/template). The rollup configuration defaults to dev mode.

**Comment by ka1to1** [comment]
any update on this? I'm getting this with Vite

**Comment by nolimits4web** [comment]
Closing as fixed in 6.1.0 beta
